src/components/Student.jsx

```diff
@@ -38,7 +65,8 @@ const Student = () => {
                 <Card.Text>Program: {student.program}</Card.Text>
                 {
                   student.courses.map((course) => {
-                    <Badge Key={course.courseId} bg="secondary">
+                    <Badge key={course.code} bg="secondary"
+                      onClick={dropCourse(student.studentd, course.codes)}>
```

Comment:
I think this should be more like `onClick={() => dropCourse(student.studentId, ...)}`